### PR TITLE
clean up devfile env tests and uses dns supported component name 

### PR DESCRIPTION
**Pull request by girishramnani**
**What type of PR is this?**
/kind cleanup

**What does does this PR do / why we need it**:
clean up devfile env tests and uses dns supported component name 

**PR acceptance criteria**:

- [ ] Unit test 

- [X] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: tests/integration/devfile/cmd_devfile_env_test.go [hunk 1]
```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/openshift/odo/tests/helper"
 )
 
-var _ = Describe("odo devfile env command tests", func() {
+var _ = FDescribe("odo devfile env command tests", func() {
```
Comment (amitkrout) [inline tests/integration/devfile/cmd_devfile_env_test.go:13]: FDescribe -> Describe

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: tests/integration/devfile/cmd_devfile_env_test.go (hunk 1 above)
Comment (prietyc123) [inline tests/integration/devfile/cmd_devfile_env_test.go:13]: you mean Describe 🙄

**Review by prietyc123 — CHANGES_REQUESTED** [review]

File: tests/integration/devfile/cmd_devfile_env_test.go (hunk 1 above)
Comment (girishramnani) [reply tests/integration/devfile/cmd_devfile_env_test.go]: hehe, resolved

File: tests/integration/devfile/cmd_devfile_env_test.go [hunk 2]
```diff
@@ -81,26 +81,6 @@ var _ = Describe("odo devfile env command tests", func() {
 				testDebugPort,
 			}
 			helper.MatchAllInOutput(output, wantOutput)
-		})
-
-		It("Should fail to set the invalid parameter", func() {
```
Comment (amitkrout) [inline tests/integration/devfile/cmd_devfile_env_test.go:86]: Its good to keep a negative case for validation.

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: tests/integration/devfile/cmd_devfile_env_test.go (hunk 2 above)
Comment (prietyc123) [reply tests/integration/devfile/cmd_devfile_env_test.go:86]: Its already there https://github.com/openshift/odo/pull/3793/files#diff-e3308315be7824d6d6b338513357491bR94

Comment (amitkrout) [reply tests/integration/devfile/cmd_devfile_env_test.go:86]: Ahh...found it
https://github.com/openshift/odo/pull/3793/files#diff-e3308315be7824d6d6b338513357491bR96

**Comment by prietyc123** [comment]
looks good to me 

/lgtm

File: tests/integration/devfile/cmd_devfile_env_test.go (hunk 1 above)
Comment (prietyc123) [reply tests/integration/devfile/cmd_devfile_env_test.go]: cool 😄

**Pull request merged**